scripts/install2

```diff
@@ -341,7 +352,6 @@ fi
 
 # Install possibly missing tools
 sudo apt-get install haveged -y || true
-sudo apt-get install iw -y || true
```

Comment:
Are you sure this isn't needed? It must have been added because one of the Armbians don't come with it... was it the Espressobin?

Reply:
its a revert to a commit done only a few days ago when we thought it didnt include it in Debian, turned out to be a path problem. Its just a correction